### PR TITLE
Add 1st Implementation Pink Edition

### PR DESCRIPTION
**Pull request by valescaf**
I would like to add a Pink Edition #hacktoberfest

### CODE REVIEW

**Comment by lucianopf** [comment]
Nice one!! :eyes: :clap: 

Would you be able to provide a gif of your solution working? :grimacing: 

When viewing it from Github its even nicer if there's a gif at the README.md file! :grimacing: :heart_eyes:

File: playground/1st-implementation-with-tables-pinkedition/README.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<p align="center">
+  <a href="https://filipedeschamps.github.io/doom-fire-algorithm/playground/1st-implementation-with-tables/">
+    <img src="https://github.com/filipedeschamps/doom-fire-algorithm/blob/master/doom-fire.gif?raw=true" width="490">
+  </a>
```
Comment (filipedeschamps) [inline playground/1st-implementation-with-tables-pinkedition/README.md]: Is pointing to the first version, not yours

File: playground/1st-implementation-with-tables-pinkedition/README.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+</p>
+
+# Doom Fire Algorithm
+Fire effect from Doom implemented in plain JavaScript and using a Table to render the final effect.
+
+- [Click here for the Demo page](https://filipedeschamps.github.io/doom-fire-algorithm/playground/1st-implementation-with-tables/)
```
Comment (filipedeschamps) [inline playground/1st-implementation-with-tables-pinkedition/README.md]: This one is also pointing to the first

File: playground/1st-implementation-with-tables-pinkedition/README.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+- [Click here for the Video tutorial](https://www.youtube.com/watch?v=HCjDjsHPOco)
+
+This project was inspired by [this article](http://fabiensanglard.net/doom_fire_psx/).
+
+## Author
+
+| [<img src="https://avatars0.githubusercontent.com/u/4248081?v=3&s=115"><br><sub>@filipedeschamps</sub>](https://github.com/filipedeschamps) |
```
Comment (filipedeschamps) [inline playground/1st-implementation-with-tables-pinkedition/README.md]: No, you are the author :)

**Comment by valescaf** [comment]
@filipedeschamps  I'd did the resquested modifications.
@lucianopf a gif is a nice feature, I'm working on that.

File: playground/1st-implementation-with-tables-pinkedition/README.md (hunk 2 above)
Comment (filipedeschamps) [reply playground/1st-implementation-with-tables-pinkedition/README.md]: The end url will be `https://filipedeschamps.com.br/doom-fire-algorithm/playground/1st-implementation-with-tables-pinkedition/`

**Comment by filipedeschamps** [comment]
I will merge the way it is and make small adjustments. You can send the gif later :)

**Pull request merged**